### ISSUE TITLE
Move all relevant Python PLN unit tests to new PLN

### ISSUE THREAD

**Issue by AmeBel** [opencog/opencog#1549]

**Comment by linas** [comment]
Can this be closed? Is it done yet?

**Comment by AmeBel** [comment]
@ngeiswei please comment

**Comment by ngeiswei** [comment]
No, they haven't been ported. They might never be, but I guess it doesn't hurt to let it open for now.

**Comment by ngeiswei** [comment]
In case someone wants to go on with porting these examples they may be found here https://github.com/opencog/opencog/releases/tag/obsolete-Python-PLN
